src/util/nextThought.js

```diff
@@ -241,6 +240,15 @@ export const nextThought = path => {
   const { value, rank } = head(path)
   const rankedContext = rootedContextOf(path)
   const contextChain = splitChain(path, contextViews)
+  const context = pathToContext(rankedContext)
+
+  if (isValidContextView(rankedContext) || isValidContextView(path)) {
+    return nextInContextView(value, rank, path, rankedContext, contextChain)
+  }
+  else {
+  // not a valid context view. Try navigating thoughts
+    return nextInThoughtView(value, context, rank, path, contextChain)
```

Comment:
Please use the ternary operator to express both cases as a single return statement.